conda_recipe_manager/parser/_types.py

```diff
@@ -132,6 +132,11 @@ class Regex:
     # Finds `environ[]` used by a some recipe files. Requires a whitespace character to prevent matches with
     # `os.environ[]`, which is very rare.
     PRE_PROCESS_ENVIRON: Final[re.Pattern[str]] = re.compile(r"\s+environ\[(\"|')(.*)(\"|')\]")
+
+    # Finds `environ.get` which is used in a closed source community of which the author of this comment
+    # participates in.
```

Comment:
I really, really appreciate having context comments. I'll just amend this to include the PR.
```suggestion
    # participates in. See Issue #271 for more details.
```